### PR TITLE
Add clean_hour script

### PR DESCRIPTION
**Pull request by nriss**

### CODE REVIEW

**Comment by nriss** [comment]
This script is needed because date and hour are in different columns in some case.
That's why we have to manage them separately and then to merge the date and the hour

File: scripts/custom/clean_hour.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+def clean_hour(raw_input):
+    """Convert 800, 1400, 1715 to 08:00, 14:00, 17.15
+    """
+    if not isinstance(raw_input, str):
+        raw_input = str(raw_input)
+    if utils.is_empty(raw_input):
+        return ""
+
+    hour = raw_input
+    while len(hour) < 4:
```
Comment (simonvadee) [inline scripts/custom/clean_hour.py:13]: there is a function on string to do that: https://www.tutorialspoint.com/python/string_rjust.htm

**Review by simonvadee — CHANGES_REQUESTED** [review]

File: scripts/custom/clean_hour.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
+from scripts import utils
+
+
+def clean_hour(raw_input):
+    """Convert 800, 1400, 1715 to 08:00, 14:00, 17.15
```
Comment (Jasopaum) [inline scripts/custom/clean_hour.py:5]: 17.15 -> 17:15

**Review by Jasopaum — COMMENTED** [review]
Can we also verify the input format?
What if the input string doesn't have only numbers? What if has more than 4 characters?
As Simon said, maybe using datetime would be safer

File: scripts/custom/clean_hour.py (hunk 1 above)
Comment (nriss) [reply scripts/custom/clean_hour.py:13]: so cool this function

**Pull request closed without merging**